.github/issue_template.md

```diff
@@ -0,0 +1,14 @@
+### Steps to reproduce
+
+(Provide either a sample application or create an executable test script using one of the templates [here](https://github.com/rails/rails/tree/master/guides/bug_report_templates))
```

Comment:
how about if we point to the actual guides page. http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#creating-a-bug-report ?